Sources/DocCArchive/Schema_0_1/InlineContent.swift

```diff
@@ -27,6 +28,7 @@ extension DocCArchive.DocCSchema_0_1 {
           return "\(id)\(isActive ? "" : "-inactive")"
         case .image    (let id)      : return "<img \(id)>"
         case .emphasis (let content) : return "*\(content)*"
+        case .strong (let content) : return "**\(content)**"
```

Comment:
Could you please align the colons? Thanks!

Reply:
Done